### PR TITLE
US-12284 : Remove save and come back link , rename cancel button in Un-Auth journey

### PR DESCRIPTION
**Pull request by Sasankachyuta13**
As a part of this User story  , Save and come back link is been removed.

cancel button is renamed to close claim.

created a common functionality and placed it in utils , so if any un-auth related story comes in near future, we can use that functionality for conditioning.

### CODE REVIEW

File: src/components/helpers/utils.ts [hunk 1]
```diff
@@ -70,3 +70,13 @@ export const isFieldSetReqiredForSelectComponent = (label: string) => {
   const arrFieldSetNotRequiredForSelectComponent = ['name of building society'];
   return !arrFieldSetNotRequiredForSelectComponent.includes(label.toLocaleLowerCase());
 };
+
+export const isUnAuthJourney = () => {
+  const containername = PCore.getContainerUtils().getActiveContainerItemName(
+    `${PCore.getConstants().APP.APP}/primary`
+  );
+  const context = PCore.getContainerUtils().getActiveContainerItemName(`${containername}/workarea`);
+  const UnAuth = PCore.getStoreValue('.CaseType', 'caseInfo.content', context);
```
Comment (AkhilaHMRC) [inline src/components/helpers/utils.ts:79]: rename UnAuth to caseType

File: src/components/helpers/utils.ts [hunk 2]
```diff
@@ -70,3 +70,13 @@ export const isFieldSetReqiredForSelectComponent = (label: string) => {
...
+  const isUnAuthJourney = UnAuth === 'Unauth' ? true : false;
+  return isUnAuthJourney;
```
Comment (AkhilaHMRC) [inline src/components/helpers/utils.ts:81]: return caseType === 'Unauth';
no need to evaluate and assign to a const

File: src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx [hunk 3]
```diff
@@ -41,29 +43,30 @@ export default function ActionButtons(props) {
               key={sButton.actionID}
               attributes={{ type: 'button' }}
             >
-              {localizedVal(sButton.name, localeCategory)}
+              {!isUnAuth ? localizedVal(sButton.name, localeCategory) : t('CLOSE_CLAIM')}
```
Comment (AkhilaHMRC) [inline src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx:46]: isUnAuth ? t('CLOSE_CLAIM') : localizedVal(sButton.name, localeCategory)

Dont negate the condition unnecessarily, it becomes complex to read

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]

File: src/components/helpers/utils.ts (hunk 1 above)
Comment (Sasankachyuta13) [reply src/components/helpers/utils.ts]: Yes, Renamed .

File: src/components/helpers/utils.ts (hunk 2 above)
Comment (Sasankachyuta13) [reply src/components/helpers/utils.ts]: Yep, included this change.

File: src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx (hunk 3 above)
Comment (Sasankachyuta13) [reply src/components/override-sdk/infra/ActionButtons/ActionButtons.tsx]: Done.

**Review by AkhilaHMRC — APPROVED** [review]

**Pull request merged**